packages/@aws-cdk/aws-lambda-event-sources/README.md

```diff
@@ -137,6 +138,7 @@ and add it to your Lambda function. The following parameters will impact Amazon
 
 * __batchSize__: Determines how many records are buffered before invoking your lambda function - could impact your function's memory usage (if too high) and ability to keep up with incoming data velocity (if too low).
 * __startingPosition__: Will determine where to being consumption, either at the most recent ('LATEST') record or the oldest record ('TRIM_HORIZON'). 'TRIM_HORIZON' will ensure you process all available data, while 'LATEST' will ignore all records that arrived prior to attaching the event source.
+* __enabled__: Disables or enables the event source mapping to pause/start polling and invocation.
```

Comment:
Is this not applicable to other event sources, such as S3 and SNS?

The [documentation here](https://docs.aws.amazon.com/lambda/latest/dg/API_CreateEventSourceMapping.html) seems to suggest that they're all event sources, no?